plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -201,10 +215,14 @@ module.exports = class ServiceTree extends Tree {
               return;
             }
 
-            let serviceLabels = service.getLabels();
-
+            let serviceLabels;
             if (service instanceof Service) {
-              serviceLabels = ServiceUtil.convertServiceLabelsToArray(service);
+              serviceLabels = Object.keys(service.getLabels()).map(key => ({
+                key,
+                value: service.getLabels()[key]
```

Comment:
`Object.entries` seems like a good idea here: 

```typescript
serviceLabels = Object.entries(service.getLabels()).map(([key, value])=> ({key, value}));
```

Reply:
copy pasted 🤷‍♂️  I can change, if you want me to change.